Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -48,6 +48,40 @@ def test_tk_setPalette(self):
                 '^must specify a background color$',
                 root.tk_setPalette, highlightColor='blue')
 
+    def test_after_info(self):
+        root = self.root
+        # Add timer.
+        timer = root.after(1, lambda: 'break')
+
+        # With no parameter, it returns a tuple of the event handler ids.
+        self.assertEqual(root.after_info(), (timer, ))
```

Comment:
Add a test for `after_info()` when no events are waiting.

Reply:
Done

Reply:
Actually I'm not sure how robust this test. What if `after_info()` return events generated in other tests? Or generated internally by Tcl machinery? I don't know if this is an issue.

Reply:
I thought about that, but I also thought that each test should be isolated and repeatable, so with that, shouldn't it be ok?  I can add a cleanup or teardown to iterate through all the event handlers and after_cancel on each one to make sure it's empty?